packages/insight/src/components/search.tsx

```diff
@@ -163,7 +164,7 @@ const Search: FC<SearchProps> = ({borderBottom, id, setErrorMessage}) => {
   return (
     <SearchForm onSubmit={search} borderBottom={borderBottom}>
       <span style={{display: 'flex', alignItems: 'center' }}>
-        <img src={searchIcon} alt='Search' style={{padding: 7}}></img>
+        <img src={searchIcon} alt='Search' style={{padding: 7, marginLeft: isMobile() && currency ? '-8px': 0}}></img>
```

Comment:
I would avoid mixing passing function calls and ternaries like this, it's a bit cleaner to just evaluate this before the component return and use that. 

You could also extract this out to its own styled component:
```
const searchImg = styled.img``;
```